src/arbeidssøkerskjema/side/Side.tsx

```diff
@@ -59,7 +59,7 @@ const Side: React.FC<ISide> = ({
             {children}
           </main>
         </Panel>
-        {!kommerFraOppsummering && !innsending && (
+        {!kommerFraOppsummering && !innsending && nesteRoute && (
```

Comment:
Noen spørsmål rundt disse - for en som ikke kjenner bakgrunnen:

`kommerFraOppsummering` -> Betyr dette at brukeren er ferdig og har sendt inn skjemaet, eller er det bare å gå tilbake?

`innsending` er boolsk - betyr det at det er sendt inn eller at det holder på å sendes inn?

Vurder å trekke ut disse tre til en variabel som har et funksjonelt navn - `skalViseKnapper`

Reply:
`kommerFRaOppsummering` er når brukeren er på Oppsummeringssiden og trykker "endre informasjon". Det er for at vi skal kunne vise "Til oppsummerings"-knappen i stedet for at brukeren skal måtte trykke "neste" gjennom alle steg før de dukker opp tilbake på oppsummering. Denne er litt overflødig siden vi i skjemaet kun har ett spørsmålsteg, men det ble en diskusjon om det igår når vi så på appen med de funksjonelle, og Tuva mente at det var greit at man eksplisitt sa "Endre informasjon" enn at bruker ble usikker på om man kom itl å miste data hvis man trakk "tilbake" :) 

Bra påpekt! Endrer navn :)